tests/snapshots/tests.contrib.graphene.test_graphene.test_schema_failing_execute_v2.json

```diff
@@ -0,0 +1,76 @@
+[[
+  {
+    "name": "graphql.query",
+    "service": "graphql",
+    "resource": "query something{ patron { id name age } }",
+    "trace_id": 0,
+    "span_id": 1,
+    "parent_id": 0,
+    "type": "graphql",
+    "error": 1,
+    "meta": {
+      "_dd.p.dm": "-0",
+      "error.msg": "exception was raised in a graphene query",
+      "error.type": "graphql.error.base.GraphQLError",
+      "runtime-id": "926b4e2492e04fd48792859f6506c822"
+    },
+    "metrics": {
+      "_dd.agent_psr": 1.0,
+      "_dd.measured": 1,
+      "_dd.top_level": 1,
+      "_dd.tracer_kr": 1.0,
+      "_sampling_priority_v1": 1,
+      "system.pid": 34633
+    },
+    "duration": 4591000,
+    "start": 1659552018079872000
+  },
+     {
+       "name": "graphql.parse",
+       "service": "graphql",
+       "resource": "graphql.parse",
+       "trace_id": 0,
+       "span_id": 2,
+       "parent_id": 1,
+       "type": "graphql",
+       "duration": 214000,
+       "start": 1659552018080037000
+     },
+     {
+       "name": "graphql.validate",
+       "service": "graphql",
+       "resource": "graphql.validate",
+       "trace_id": 0,
+       "span_id": 3,
+       "parent_id": 1,
+       "type": "graphql",
+       "duration": 1407000,
+       "start": 1659552018080321000
+     },
+     {
+       "name": "graphql.execute",
+       "service": "graphql",
+       "resource": "query something{ patron { id name age } }",
+       "trace_id": 0,
+       "span_id": 4,
+       "parent_id": 1,
+       "type": "graphql",
+       "error": 1,
+       "meta": {
+         "error.msg": "exception was raised in a graphene query",
```

Comment:
are these suppose to be so bare/useless?

Reply:
Do you mean the error message? I am setting that error message here: https://github.com/DataDog/dd-trace-py/blob/90ca0bcc7e9a9bd9f25efe33b7fd8101042fcf0f/tests/contrib/graphene/test_graphene.py#L26